### ISSUE TITLE
github.com/auth0/webtask-json-validator no longer exists

### ISSUE THREAD

**Issue by zikes** [auth0-extensions/auth0-delegated-administration-extension#176]
```sh
$ npm install
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/auth0/webtask-json-validator.git
npm ERR! 
npm ERR! ERROR: Repository not found.
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/zikes/.npm/_logs/2019-04-12T20_00_21_139Z-debug.log
```

**Comment by shawnmclean** [comment]
Hi @zikes, this should be fixed by #169 that is merging soon.

**Comment by shawnmclean** [comment]
#169 merged in, master should now use the right dependencies.
